concepticondata/conceptlists/Ferguson-1964-30.tsv

```diff
@@ -3,22 +3,22 @@ Ferguson-1964-30-1	1	KIN	mother	1216	MOTHER
 Ferguson-1964-30-2	2	KIN	father	1217	FATHER
 Ferguson-1964-30-3	3	KIN	baby	1289	BABY
 Ferguson-1964-30-4	4	BODY	food	256	FOOD
-Ferguson-1964-30-5	5	BODY	drink, water	1401	DRINK
+Ferguson-1964-30-5	5	BODY	drink, water		
 Ferguson-1964-30-6	6	BODY	sleep	1585	SLEEP
 Ferguson-1964-30-7	7	BODY	urination	592	PISS
-Ferguson-1964-30-8	8	BODY	defecation		
+Ferguson-1964-30-8	8	BODY	defecation	676	SHIT (DEFECATE)
```

Comment:
I would argue against it because of inconsistency in the POS of the elicitation gloss and the concept set. Same goes for 'urination' mapped to PISS.

Reply:
The problem with this list is that it elicits special vocabulary that adults use towards babies.

So the English words that were elicited by _urination_ and _defecation_, were _wee-wee_ and _pee-pee_, and _poop(oo)_, respectively. Difficult to decide, but I would say they refer to the process mainly and should therefore be mapped to the verb-concepts.

But we can also leave them completely unmapped.

Reply:
I'd leave both unmapped. If I think of parents talking to their children they often say/ask something like "you made poo-poo or pee-pee".

Reply:
The elicitation glosses clearly can't refer to the concrete nouns.

But I agree that it's probably better to unmap them to err on the save side.